modules/webbrowser/tests/regression/webbrowser/script.py

```diff
@@ -0,0 +1,13 @@
+# Inviwo Python script 
+import inviwopy
+import ivw.regression
+import time
+
+network = inviwopy.app.network;
+isLoading = network.Webbrowser.isLoading;
+# HACK: Wait for as little as possible while ensuring that the webpage has re-rendered. 
+while isLoading.value:
+	time.sleep(0.5);	
```

Comment:
Could probably decrease this interval now

Reply:
This was chosen to be on the safe side. Redraw interval of CEF (1/30s) + one or more network evaluations.